tests/playwright/sanity/core/base/document.test.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+
+test.describe( 'Document tests', async () => {
+	test( 'converting gutenberg page to sections columns',
+		async ( { page }, testInfo ) => {
+			const wpAdmin = new WpAdminPage( page, testInfo );
+			await wpAdmin.setExperiments( {
+				container: 'inactive',
+			} );
+
+			await wpAdmin.page.goto( '/wp-admin/post-new.php?post_type=page' );
+			await addElement( wpAdmin, 'list' );
+			await addElement( wpAdmin, 'heading' );
+			const editor = await wpAdmin.convertFromGutenberg();
+			const sections = await editor.getPreviewFrame().locator( '[data-element_type="section"]' ).count();
+			await expect( sections ).toEqual( 1 );
+			const columns = await editor.getPreviewFrame().locator( '[data-element_type="column"]' ).count();
+			await expect( columns ).toEqual( 1 );
+			const textEditors = await editor.getPreviewFrame().locator( '.elementor-widget-text-editor' ).count();
+			await expect( textEditors ).toEqual( 1 );
+
+			await wpAdmin.setExperiments( {
+				container: 'active',
+			} );
+		} );
+
+	test( 'converting gutenberg page to container',
+		async ( { page }, testInfo ) => {
+			const wpAdmin = new WpAdminPage( page, testInfo );
+			await wpAdmin.setExperiments( {
+				container: 'active',
+			} );
+
+			await wpAdmin.page.goto( '/wp-admin/post-new.php?post_type=page' );
+			await addElement( wpAdmin, 'list' );
+			await addElement( wpAdmin, 'heading' );
+			const editor = await wpAdmin.convertFromGutenberg();
+			const sections = await editor.getPreviewFrame().locator( '[data-element_type="container"]' ).count();
+			await expect( sections ).toEqual( 1 );
+			const textEditors = await editor.getPreviewFrame().locator( '.elementor-widget-text-editor ' ).count();
+			await expect( textEditors ).toEqual( 1 );
+
+			await wpAdmin.setExperiments( {
+				container: 'inactive',
+			} );
+		} );
+} );
+
+async function addElement( wpAdmin: WpAdminPage, elementType: string ) {
+	await wpAdmin.page.click( '.block-editor-inserter__toggle' );
```

Comment:
<img width="665" alt="Screenshot 2023-07-27 at 18 14 16" src="https://github.com/elementor/elementor/assets/94352322/491cad4b-5ed7-454d-a294-72fbc4cb1392">

Do you know why I get this warning?

Reply:
overlay is blocking the button ?